app/experimenter/experiments/tests/test_models.py

```diff
@@ -2176,6 +2207,18 @@ def test_launch_to_live_is_not_considered_latest_rejection(self):
 
         self.assertIsNone(experiment.changes.latest_rejection())
 
+    def test_update_live_is_not_considered_latest_rejection(self):
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.LIVE_APPROVE_APPROVE,
+        )
+
+        experiment.status = NimbusExperiment.Status.LIVE
+        experiment.publish_status = NimbusExperiment.PublishStatus.IDLE
+        experiment.save()
+        generate_nimbus_changelog(experiment, experiment.owner, "test message")
```

Comment:
Shouldn't it already be in this state with that lifecycle?

Reply:
Whoops! Yup 👍